core/src/com/sonyericsson/chkbugreport/plugins/logs/event/EventLogPlugin.java

```diff
@@ -353,7 +353,7 @@ private void addSampleData(Module br, String eventType, LogLine sl) {
             String name = null;
             if (eventType.equals("dvm_lock_sample")) {
                 // This is a bit different
-                duration = Integer.parseInt(sl.fields[fieldCount-2]);
+                duration = Integer.parseInt(sl.fields[3]);
```

Comment:
Does this also need to be done for some of the other indexes here? Also is the fieldCount needed at all?

Reply:
We could do it with all of them... I dont' really see the point, although there is a check like so that seems reasonable:

```
if(fieldCount < 4)
```

Shall I update them and just use `sl.fields.length` in the `if` instead?

Reply:
Sadly no we can't refactor this, we'd need to fix the parsing/splitting in LogLine as well.  Currently it ends up with different numbers of fields and these are tracked from the end based on where the spaces end up.  Its... less than optimal.